userspace/libscap/scap_vtable.h

```diff
@@ -38,9 +38,9 @@ typedef struct ppm_evt_hdr scap_evt;
 #define PPM_IOCTL_MASK_UNSET_EVENT _IO(PPM_IOCTL_MAGIC, 7)
  */
 enum scap_eventmask_op {
-	SCAP_EVENTMASK_ZERO = 0x7305, //< disable all events
-	SCAP_EVENTMASK_SET = 0x7306, //< enable an event
-	SCAP_EVENTMASK_UNSET = 0x7307, //< disable an event
+	SCAP_PPM_SC_MASK_ZERO = 0x7305, //< disable all syscalls
+	SCAP_PPM_SC_MASK_SET = 0x7306, //< enable a syscall
+	SCAP_PPM_SC_MASK_UNSET = 0x7307, //< disable a syscall
```

Comment:
I don't remember if we rely on these values to save a bit of work in kmod, but they're outdated now (they refer to the just-removed ioctls)

Not sure if we want to do something about it, your call